src/frontend/components/DashboardThumbnail/index.spec.tsx

```diff
@@ -2,30 +2,53 @@ import { act, fireEvent, render, screen } from '@testing-library/react';
 import fetchMock from 'fetch-mock';
 import React from 'react';
 
+import {
+  UploadManagerContext,
+  UploadManagerStatus,
+} from 'components/UploadManager';
+import { useThumbnail } from 'data/stores/useThumbnail';
+import { modelName } from 'types/models';
+import { uploadState } from 'types/tracks';
+import { Deferred } from 'utils/tests/Deferred';
+import { thumbnailMockFactory, videoMockFactory } from 'utils/tests/factories';
+import { wrapInIntlProvider } from 'utils/tests/intl';
 import { DashboardThumbnail } from '.';
-import { useThumbnail } from '../../data/stores/useThumbnail';
-import { modelName } from '../../types/models';
-import { uploadState } from '../../types/tracks';
-import { Deferred } from '../../utils/tests/Deferred';
-import { videoMockFactory } from '../../utils/tests/factories';
-import { wrapInIntlProvider } from '../../utils/tests/intl';
-import { UploadManagerContext, UploadManagerStatus } from '../UploadManager';
+import WS from 'jest-websocket-mock';
+import { initVideoWebsocket } from 'data/websocket';
```

Comment:
these two dependencies are not ordered

Reply:
Indedd, I move to move `jest-websocket-mock` to the first block